draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1117,34 +1117,42 @@ modification at a later date. Note that as per [RFC7231], in the case of a 503
 response the log MAY include a `Retry-After:` header in order to request a
 minimum time for the client to wait before retrying the request.
 
-## Add Chain to Log    {#add-chain}
+## Add Entry to Log    {#add-entry}
 
-POST https://\<log server>/ct/v2/add-chain
+POST https://\<log server>/ct/v2/add-entry
 
 Inputs:
 
-: chain:
-  : An array of base64 encoded certificates. The first element is the
-    certificate for which the submitter desires an SCT; the second certifies the
+: precertificate:
+  : The base64 encoded precertificate.
+
+  certificate:
+  : The base64 encoded certificate.
+
```

Comment:
I suggest adding a note here to say that either `precertificate` or `certificate` needs to be supplied.  Not both, and not neither.